python/GangaLHCb/Lib/Applications/BenderBox.py

```diff
@@ -0,0 +1,520 @@
+#  @author Vanya BELYAEV  Ivan.Belyaev@itep.ru       
+class BenderModule(GangaObject):
+    """Helper class to define the main properties of Bender applicatiom
+    - the name of Bender module to run
+    - dictionary of parameters to be forwarder to <code>configure</code> method
+    - number of event to process
+
+    ======
+    Usage:
+    ======
+    
+    j.application = BenderBox ( tool = BenderModule ( module = 'the_path/the_module.py' , events = 1000 ) )
+
+    """    
+    _schema = Schema(Version(1, 0), {
+        'module'    : FileItem   (
+        defvalue  = File () ,
+        doc       = """The file with Bender module. It is expected that module contains methods ``configure'' & ``run'' with the proper signatures""") , 
+        'params'    : SimpleItem (
+        defvalue  = {} ,
+        typelist  = ['dict', 'str', 'int', 'bool', 'float'],
+        doc       = """The dictionary of parameters to be forwarded to ``configure'' method of the supplied Bender module""") , 
+        'events'    : SimpleItem ( defvalue=-1, typelist=['int'], doc= "Number of events to process"),
+        })
+    _category = 'BenderTool'
+    _name     = 'BenderModule'
+    
+    layout    = """#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# =============================================================================
+importOptions('{datafile}')
+from Gaudi.Configuration import EventSelector,FileCatalog
+from copy import deepcopy
+inputdata    = deepcopy( EventSelector() .Input    )
+filecatalogs = deepcopy( FileCatalog  () .Catalogs )
+EventSelector().Input    = [] ## reset it 
+FileCatalog  ().Catalogs = [] ## reset it 
+# =============================================================================
+# configure Bender and run it 
+# =============================================================================
+import  sys
+sys.path = sys.path + ['.']
+import {modulename} as USERMODULE
+USERMODULE.configure( inputdata , filecatalogs {paramstring})
+USERMODULE.run({events})
+# =============================================================================
+# The END
+# =============================================================================
+"""
+
+    ## Returns file names which should be copyed to shared area
+    def inputfiles(self):
+        "Returns file names which should be copyed to shared area"
+        return [LocalFile(self.module.name),]
+
+    ## Returns  wrapper file which run at WN
+    def getWrapper(self, data_file):
+        " Returns  wrapper file which run at WN"
+        
+        module_name  = os.path.split(self.module.name)[-1].split('.')[0]
+        param_string =  ',params=%s' % self.params if self.params else ''
+        
+        the_script      = self.layout.format (
+            datafile    = data_file ,
+            modulename  = module_name ,
+            paramstring = param_string ,
+            events      = self.events
+            )
+
+        from GangaLHCb.Lib.RTHandlers.RTHUtils import getXMLSummaryScript
```

Comment:
To be consistent with what we've tried to move to with regard to code standards across the project. imports should always be done at the global level in the file unless there's a good reason it can't in python (at least from numerous discussions about standards I've been told this is the case).